packages/cozy-device-helper/src/flagship.ts

```diff
@@ -8,46 +8,34 @@ export enum FlagshipRoutes {
   Stack = 'stack'
 }
 
-export enum BiometryType {
-  Face = 'Face',
-  FaceID = 'FaceID',
-  Fingerprint = 'Fingerprint',
-  Iris = 'Iris',
-  TouchID = 'TouchID'
-}
+export type BiometryType = 'TouchID' | 'FaceID' | 'Biometrics'
 
 export interface FlagshipMetadata {
-  capabilities?: {
-    biometryType?: BiometryType
-  }
-  hasBiometry?: boolean
+  biometry_available?: boolean
+  biometry_type?: BiometryType
   immersive?: boolean
   navbarHeight?: number
   platform?: Record<string, unknown>
   route?: FlagshipRoutes
-  settings?: {
-    PINEnabled?: boolean
-    autoLockDelay?: number
-    autoLockEnabled?: boolean
-    biometryEnabled?: boolean
-  }
+  settings_PINEnabled?: boolean
+  settings_autoLockDelay?: number
```

Comment:
nit: I should have commented that in previous PRs, but it would be nice to have the unit in the variable name (i.e `settings_autoLockDelayMs`)

Reply:
There will most likely be another release soon